src/aics-image-viewer/components/App/index.tsx

```diff
@@ -277,6 +277,16 @@ const App: React.FC<AppProps> = (props) => {
   playControls.onPlayingAxisChanged = (axis) => {
     loader.current?.setPrefetchPriority(axis ? [axisToLoaderPriority[axis]] : []);
     loader.current?.syncMultichannelLoading(axis ? true : false);
+    if (image) {
+      if (axis === null) {
+        // Playback has stopped - reset scale level bias
+        view3d.setScaleLevelBias(image, 0);
+      } else {
+        // Playback has started - decide whether to lower the scale level to speed up loading
+        const shouldDownlevel = axis === "t" || numSlices[axis] !== numSlicesLoaded[axis];
```

Comment:
so we should downlevel if the dimension of the playback axis changed?  Not sure I understand when that condition arises. Maybe the comment should say why, too...

Reply:
"if the axis we're playing through is time, we should always downlevel; if it's spatial, we should only downlevel if we don't already have the entire dimension in memory." This is to prevent an unnecessary load if we're playing through x or y, e.g. I'll add that as a comment.